testing/src/main/java/io/stargate/it/http/RestApiExtension.java

```diff
@@ -228,6 +230,14 @@ private Instance(StargateEnvironmentInfo stargateEnvironmentInfo, RestApiParamet
         cmd.addArgument("-D" + e.getKey() + "=" + e.getValue());
       }
 
+      if (isDebug()) {
+        int debuggerPort = 5200;
+        cmd.addArgument(
+            "-agentlib:jdwp=transport=dt_socket,server=n,suspend=y,"
+                + "address=localhost:"
+                + debuggerPort);
+      }
```

Comment:
A bit off-topic, but this is nice to have as it allows debugging the REST server in integration tests (as explained [here](https://github.com/stargate/stargate/blob/master/DEV_GUIDE.md#debugging-integration-tests)).

Reply:
cool!